k8s_snapshots/backends/google.py

```diff
@@ -122,10 +124,23 @@ def get_disk_identifier(volume: pykube.objects.PersistentVolume) -> GoogleDiskId
     gce_disk_zone = volume.labels.get(
         'failure-domain.beta.kubernetes.io/zone'
     )
+
     if not gce_disk_zone:
         raise UnsupportedVolume('cannot find the zone of the disk')
 
-    return GoogleDiskIdentifier(name=gce_disk, zone=gce_disk_zone)
+    gce_disk_region = volume.labels.get(
+        'failure-domain.beta.kubernetes.io/region'
+    )
+
+    if not gce_disk_region:
+        raise UnsupportedVolume('cannot find the region of the disk')
+
+    if "__" in gce_disk_zone:
+        # seems like Google likes to put __ in between zones in the label
+        # failure-domain.beta.kubernetes.io/zone when the pv is regional
+        return GoogleDiskIdentifier(name=gce_disk, zone="N/A", region=gce_disk_region, regional=True)
```

Comment:
does `"N/A"` have a particular meaning here, or can we replace this with `None` to avoid confusion later? or better yet, just don't set it. you can make the `zone` and `region` args optional [above](https://github.com/farmersedgeinc/k8s-snapshots/pull/1/files#diff-4833b94ee634df43434ac52259ae24b3R107-R108) by adding `= none` to the end of those lines

Reply:
although, if the original author has decided to give special meaning to the string `"N/A"` then may as well go with the flow i guess

Reply:
The original author did not use the string "N/A", I could use None instead